gridcapa-core-cc-app/src/main/java/com/farao_community/farao/gridcapa_core_cc/app/postprocessing/FileExporterHelper.java

```diff
@@ -182,99 +199,116 @@ public void exportCneToMinio(final InternalCoreCCRequest coreCCRequest) throws I
         LOGGER.info("Core CC task: '{}', creating Metadata result for timestamp: '{}'", coreCCRequest.getId(), hourlyRaoResult.getRaoRequestInstant());
         final HourlyRaoRequest hourlyRaoRequest = coreCCRequest.getHourlyRaoRequest();
-        final String metaDataFileName = NamingRules.generateMetadataFileName(hourlyRaoResult.getRaoRequestInstant(), coreCCRequest);
+        final String metaDataFileName = generateMetadataFileName(hourlyRaoResult.getRaoRequestInstant(), coreCCRequest);
 
         try (final ByteArrayOutputStream outputStreamMetaData = new ByteArrayOutputStream()) {
             final String metaDataFilePath = buildFilePath(hourlyRaoRequest.getResultsDestination(), metaDataFileName);
             final CoreCCMetadata metadata = new CoreCCMetadata(coreCCRequest.getRaoRequest().getFilename(),
-                    coreCCRequest.getRequestReceivedInstant().toString(),
-                    coreCCRequest.getHourlyRaoResult().getRaoRequestInstant(),
-                    coreCCRequest.getHourlyRaoResult().getComputationStartInstant().toString(),
-                    coreCCRequest.getHourlyRaoResult().getComputationEndInstant().toString(),
-                    coreCCRequest.getTimeInterval(),
-                    coreCCRequest.getCorrelationId(),
-                    coreCCRequest.getHourlyRaoResult().getStatus().toString(),
-                    coreCCRequest.getHourlyRaoResult().getErrorCodeString(),
-                    coreCCRequest.getHourlyRaoResult().getErrorMessage(),
-                    coreCCRequest.getVersion());
+                                                               coreCCRequest.getRequestReceivedInstant().toString(),
+                                                               hourlyRaoResult.getRaoRequestInstant(),
+                                                               hourlyRaoResult.getComputationStartInstant().toString(),
+                                                               hourlyRaoResult.getComputationEndInstant().toString(),
+                                                               coreCCRequest.getTimeInterval(),
+                                                               coreCCRequest.getCorrelationId(),
+                                                               hourlyRaoResult.getStatus().toString(),
+                                                               hourlyRaoResult.getErrorCodeString(),
+                                                               hourlyRaoResult.getErrorMessage(),
+                                                               coreCCRequest.getVersion());
             new ObjectMapper().writeValue(outputStreamMetaData, metadata);
-            minioAdapter.uploadOutputForTimestamp(metaDataFilePath, new ByteArrayInputStream(outputStreamMetaData.toByteArray()), CORE_CC, "METADATA", coreCCRequest.getTimestamp());
+            minioAdapter.uploadOutputForTimestamp(metaDataFilePath,
```

Comment:
Why do you put a newline after each parameter here while you did not previously, for instance on lines 209 & 220?